examples/hello/BUILD.bazel

```diff
@@ -0,0 +1,65 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("//tools:cdk8s.bzl", "cdk8s_import", "cdk8s_synth")
+load("//tools:defaults.bzl", "jest_test", "ts_library")
+
+cdk8s_import(
+    name = "gen",
+    path = "examples/hello/imports",
```

Comment:
feels like this should be a relative path, and I we should also make `./imports` the default, so basically I'd expect this to be omitted.

Reply:
I'll have to check to see if we can do that, but the reason is that we need to provide a relative path so that the CLI knows where to output to relative to the workspace root. Again, we may be able to get around this, but I'll have to double check.

Reply:
Turns out we can do it with a bit of bash magic 😄